src/qililab/pulse/pulse_shape/rectangular.py

```diff
@@ -28,7 +28,7 @@
 class Rectangular(PulseShape):
     """Rectangular/square pulse shape."""
```

Comment:
Not keen on visualization, the maths should be enough, I would add visualization if we had propietary good images, otherwise this should be fine for now.